generation/WinSDK/Partitions/Dxgi/settings.rsp

```diff
@@ -13,11 +13,3 @@ _D3DCOLORVALUE
 <IncludeRoot>/um/dxprogrammablecapture.h
 --namespace
 Windows.Win32.Graphics.Dxgi
---with-attribute
```

Comment:
Inversely, could we also have all the `=AssociatedEnum` in here? It seems much nicer to me to have configuration in the namespace-specific `settings.rsp` than in one humongous global file, if possible?

Reply:
No. The scraper and the emitter have different roles and supporting toolchains.

Reply:
https://github.com/microsoft/win32metadata/blob/main/docs/architecture.md

Reply:
@mikebattista unfortunately that doesn't explain directly which flags are available, nor what precedence I should stick to. Searching further in the repo shows that `--enumMakeFlags` is a feature of `ClangSharpSourceToWinmd` so it makes sense that that only works via emitter settings.

Then the question remains: do we prefer to set the `Flags` attribute on the _emitter_ via `--enumMakeFlags`, or do we prefer to set it _earlier_ via `--with-attribute` on the _scraper_ as done before this PR?